src/srcom/podiums.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+type Runs struct {
+	Data []Run
+}
+
+type Run struct {
+	Place int
+}
+
+func main() {
+	if len(os.Args) != 2 {
+		Usage("podiums", "[PLAYER NAME]", "whatevermarco")
+	}
+
+	uid, err := UserID(os.Args[1])
+	if err != nil {
+		ErrorAndDie(err)
+	}
+
+	res, err := http.Get(API + "/users/" + uid + "/personal-bests")
+	if err != nil {
+		ErrorAndDie(err)
+	}
+	defer res.Body.Close()
```

Comment:
The `Request()` function from `utils.go` should probably be used here since it also handles rate limiting.